config/locales/metadata/en.yml

```diff
@@ -508,6 +508,10 @@ en:
         data_deletion_period:
           label: Data deletion period
 
+        contaminated_human_data_access_group:
+          label: Contaminated Human Data Access Group
+          help: "Allows the specified Unix groups and users access to data separated from out from the main data product as likely contaminated with human - typically rarely used and only for validating the separation process as we may not have the ethical or legal."
+
```

Comment:
"separated from out from" -> "separated from" ?
"contaminated with human" -> "contaminated with human data" OR "contaminated with human sequence data" ?

Reply:
I copied past the text in the text, that they requested to show as helper, well done for spotting this